C/cli/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cli"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cli/cli.git", "b2e36a0979a06b94bf364552a856c166cd415234")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd cli/
+make
+mkdir ${bindir}
+mv ./bin/gh ${bindir}/gh
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
Why not FreeBSD?

Reply:
https://github.com/cli/cli/releases/tag/v1.0.0 they don't build on FreeBSD

Reply:
An error more concrete than "doesn't build"?  All go packages we have here build flawless for all platforms.

Reply:
What I meant is that 1. they don't have release for FreeBSD in the official repo, and that:
```
[09:05:49] ../../.gopath/pkg/mod/github.com/muesli/termenv@v0.6.0/termenv_unix.go:30:12: undefined: termStatusReport
[09:05:49] ../../.gopath/pkg/mod/github.com/muesli/termenv@v0.6.0/termenv_unix.go:52:12: undefined: termStatusReport
[09:05:49] ../../.gopath/pkg/mod/github.com/muesli/termenv@v0.6.0/termenv_unix.go:76:9: readfds.Bits undefined (type syscall.FdSet has no field or method Bits)
[09:05:49] ../../.gopath/pkg/mod/github.com/muesli/termenv@v0.6.0/termenv_unix.go:87:12: readfds.Bits undefined (type syscall.FdSet has no field or method Bits)
[09:05:52] make: *** [Makefile:30: bin/gh] Error 2
```

Reply:
I guess the patch in https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=248599 might help

Reply:
how should I try to apply this patch for binary builder? some `sed`?